src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -202,7 +202,7 @@
   <data name="VideoConference_Shortcuts.Header" xml:space="preserve">
     <value>Shortcuts</value>
   </data>
-  <data name="VideoConference_CameraOverlayImageAlt.AutomationProperties.Name" xml:space="preserve">
+  <data name="VideoConference_CameraOverlayImageAlt.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
```

Comment:
So weird that this is needed

Reply:
We had an email thread about this when we initially tried to move Settings to winui3. It causes crashes if not added